src/components/MobileMenu/MobileMenu.component.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Switch from '../Switch';
+import { Menu, MenuItem, SearchBar, DarkModeText } from './MobileMenu.styled';
+import { LoginButton } from '../NavBar/NavBar.styled';
+import { useGlobal } from '../../providers/GlobalContext';
+import { operationTypes } from '../../utils/stateOperations';
+
+function MobileMenu() {
+  const { state, dispatch } = useGlobal();
+  const { darkMode, menu, theme } = state;
+  const { SWITCH_THEME, SET_SEARCH } = operationTypes;
+
+  const switchTheme = () => {
+    const switchThemeAction = {
+      type: SWITCH_THEME,
+    };
```

Comment:
becasue this constants are also used on your `navbar`, you can extract them to a common file